### ISSUE TITLE
draw2dimg.SetFont doesn't work

### ISSUE THREAD

**Issue by xojoc** [llgcode/draw2d#93]
loadCurrentFont (https://github.com/llgcode/draw2d/blob/master/draw2dimg/ftgc.go#L116) doesn't check for gc.Current.Font but for gc.Current.FontData making draw2dimg.SetFont useless.

**Comment by llgcode** [comment]
it's cause gc.Current.Font is loaded against gc.Current.FontData

**Comment by EtienneBruines** [comment]
Any reason why things like `CreateStringPath` don't check for `gc.Current.Font`, and only call `gc.loadCurrentFont` if `gc.Current.Font` is not set?

**Comment by llgcode** [comment]
You're right the implementation can be optimized to check gc.Current.Font is the same as gc.Current.FontData.

**Comment by mattn** [comment]
Any update on this?
